frontend/src/components/Links/Settings/Settings.jsx

```
@@ -0,0 +1,161 @@
+    let { value } = target;
+    if (name === "target") value = target.checked;
+    setState((prev) => ({ ...prev, [name]: value }));
+  };
+
+  const handleClose = async (e) => {
+    e.preventDefault();
+    const info = Array.from(
+      settingsRef.current.querySelectorAll("input")
+    ).reduce(
+      (acc, it) => ({
+        ...acc,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)